_data/doc-nav-header.yml

```diff
@@ -18,6 +18,8 @@
       url: "/tutorials/scala-for-java-programmers.html"
     - title: Online Resources
       url: "/learn.html"
+    - title: “Hello, Scala” Book
+      url: "/overviews/hello-scala/introduction.html"
```

Comment:
Yes, I can move the link higher. I thought there might be some discussion about where to put it, so I started with it at the bottom.

I’ll also move the `hello-scala` directory up.

I added the “Book” part because I thought “Hello, Scala” in the menu looked a little strange on its own. I couldn’t think of a better way to show that this is a book. But I’ll remove it, and also remove the quotes around the Hello, Scala title.

Reply:
@lrytz I may have misread what you wrote. Are you going to change the “multipage-overview” layout formatting, or should I use the “tour” layout? Thanks! (Or if you want I can see if I can figure out the “multipage-overview” layout.)

Reply:
@julienrf I can also remove the Akka lessons, and put a link to the Akka website at the end of the Futures lesson.

Reply:
>  Are you going to change the “multipage-overview” layout formatting, or should I use the “tour” layout?

Actually, I’m not sure! My reasoning was that anyway having working “next page”/“previous page” and current page highlight in the TOC would be beneficial to other multipage overviews, so we should anyway improve that layout. I’m not sure we can easily reuse the “tour” layout without breaking things because it has been designed specifically for the Tour.

Reply:
You can try to fix the multipage-overview layout or we can do it in another PR, as you feel!

Reply:
Unfortunately I didn’t get too far on this today. I updated all my Markdown files to have next-page and previous-page links. (What I really did is write some scripts to automate changing al of the header content in the MD files.) Then as you wrote, after that the “tour” layout didn’t work, and then I *think* that the “multipage-overview” layout doesn’t work when the *hello-scala* directory is directly under the root directory. I’m pretty sure that’s the case, but I could be wrong.

I did get far enough to see that *_layouts/multipage-overview.html* has these lines:

````
<!-- TOC -->
{% include sidebar-toc-multipage-overview.html %}
````

I started to dig into *_includes/sidebar-toc-multipage-overview.html*, but that’s when I ran out of time.

If there’s someone who’s an expert on layouts I can let them work on this when they have time, otherwise I can work on it as a separate PR if you’d like, but I won’t have much time to work on it until Saturday. (Because I didn’t want to mess up other content that uses the multipage-overview layout, my initial thought was to create a new “book” layout, but it sounds like we all want next/previous links and a TOC that highlights the current page, so I talked myself out of that.)

Just let me know, I’m game either way!

Reply:
> I _think_ that the “multipage-overview” layout doesn’t work when the _hello-scala_ directory is directly under the root directory. I’m pretty sure that’s the case, but I could be wrong.

Haha, that’s the joy of working with an untyped system :joy:

Then I suggest that you leave everything under the `overview/` directory, and focus on the content of the guide.

> my initial thought was to create a new “book” layout, but it sounds like we all want next/previous links and a TOC that highlights the current page

That’s what I think too. I will take some time to improve the “multipage-overview” layout so that the current page is highlighted in the TOC, and we can have links between pages.

Reply:
Okay, sounds good, I’ll work on the content. Thanks again!

Reply:
@alvinj I’ve just created #1437 for highlighting the current page in the TOC of a multipage overview. I’m about to work on the “next”/“previous” links, but do I understand that you already did that work?

Reply:
No, I ended up not doing anything there. (When I used the Tour layout, the next/previous links worked, but everything stopped working when I moved the book content to the root directory, so I never tried to integrate that into the multipage-overview.)

Reply:
OK, I’ve created #1440 to support links between pages of an overview.